pkg/build/nodeimage/const_storage.go

```diff
@@ -25,7 +25,7 @@ NOTE: we have customized it in the following ways:
 - install as the default storage class
 */
 
-var defaultStorageImages = []string{"docker.io/rancher/local-path-provisioner:v0.0.14", "k8s.gcr.io/build-image/debian-base:buster-v1.7.2"}
+var defaultStorageImages = []string{"docker.io/rancher/local-path-provisioner:v0.0.14", "registry.k8s.io/build-image/debian-base:buster-v1.7.2"}
```

Comment:
this is hardcoded in the manifest below

Reply:
@BenTheElder we may fork the storage provisionser https://github.com/rancher/local-path-provisioner, it seems similar to kindnet, is only 3 go files

Reply:
WDYT?

Reply:
> docker.io/rancher/local-path-provisioner:v0.0.14

they are in 0.21 release and we have 0.14, also, it doesn't seem to update golang periodically

https://github.com/rancher/local-path-provisioner/commit/3436e3cf83ed0c43f3a2f92f53a5f698718d640a

it is a bit more for having a better control on this dependency we are deliverying ,